src/test/cli/rbd/help.t

```diff
@@ -420,6 +423,55 @@
     -p [ --pool ] arg    pool name
     --group arg          group name
   
+  rbd help group image add
+  usage: rbd group image add [--pool <pool>] [--group <group>] 
+                             <group-spec> <image-spec> 
+  
+  Add an image to a consistency group.
+  
+  Positional arguments
+    <group-spec>         group specification
+                         (example: [<pool-name>/]<group-name>)
+    <image-spec>         image specification
+                         (example: [<pool-name>/]<image-name>)
+  
+  Optional arguments
+    -p [ --pool ] arg    pool name
+    --group arg          group name
```

Comment:
I assume then we need to have something like: --image-pool, and --group-pool options.
Do I understand correctly that this is what you would like to see?